core/src/main/java/com/linecorp/armeria/internal/HttpHeaderUtil.java

```diff
@@ -14,19 +14,22 @@
  * under the License.
  */
 
-package com.linecorp.armeria.client;
+package com.linecorp.armeria.internal;
 
 import com.linecorp.armeria.common.util.Version;
 
 import io.netty.util.AsciiString;
 
-final class HttpHeaderUtil {
+/**
+ * Provides utility functions for internal use related with HTTP headers.
+ */
+public final class HttpHeaderUtil {
 
     private static final String CLIENT_ARTIFACT_ID = "armeria";
 
-    static final AsciiString USER_AGENT = AsciiString.cached(createUserAgentName());
+    public static final AsciiString USER_AGENT = AsciiString.cached(createUserAgentName());
 
-    static String hostHeader(String host, int port, int defaultPort) {
+    public static String hostHeader(String host, int port, int defaultPort) {
         if (port == defaultPort) {
             return host;
         }
```

Comment:
This method could be replaced with `ArmeriaHttpUtil.authorityHeader()`?

Reply:
Replaced with `ArmeriaHttpUtil.authorityHeader()`.